test cases/common/272 unity/meson.build

```diff
@@ -0,0 +1,15 @@
+project('unity', 'c',
+  default_options : [
+    'unity_size=2'])
+
+if get_option('unity') != 'on'
+  error('MESON_SKIP_TEST: unity builds not enabled')
+endif
+
+slib_notinstalled = static_library('slib_notinstalled', ['slib1.c', 'slib2.c'])
```

Comment:
If I understand correctly this is the actual test, everything after this line is not needed?

Reply:
I would also add a comment telling that you're testing a case where the number of sources is equal to unity_size.